clients/js/examples/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do we need it to be a separate package ? The go client [examples] (https://github.com/raystack/raccoon/tree/main/clients/go/examples) do not define it as a separate package. Any reason to have a separate package.

Reply:
wanted to include raystack/raccoon from npm package

Reply:
Understood